temporian/test/api_test.py

```diff
@@ -60,6 +58,46 @@ def test_evaluation(self):
                 os.path.join(tempdir, "p2.png")
             )
 
+    def test_eager_mode(self):
+        evset_1 = tp.event_set(
+            timestamps=[0.0, 2.0, 4.0, 6.0],
+            features={
+                "f1": [1.0, 2.0, 3.0, 4.0],
+                "f2": [5.0, 6.0, 7.0, 8.0],
+            },
+        )
+
+        evset_2 = tp.event_set(timestamps=[1.0, 2.0, 2.0])
+
+        h1 = tp.simple_moving_average(input=evset_1, window_length=7)
+        h2 = tp.resample(input=h1, sampling=evset_2)
+
+        # TODO: add magics to EventSet
+        # Create base class that both Node and EventSet inherit from
+        # h3 = evset_1 * 2.0 + 3.0 > 10.0
+
+        # TODO: enable __getitem__ for column selection in EventSet
+        # result = tp.glue(tp.prefix("sma_", h2["f2"]), evset_2)
+
+        result = tp.glue(tp.prefix("sma_", h2), evset_2)
```

Comment:
Can you try to "del h1" before the computation of "result" (so it looks like a potential real code).